ios/browser/api/ledger/brave_ledger.h

```diff
@@ -123,12 +115,6 @@ OBJC_EXPORT
 - (void)pendingContributionsTotal:(void (^)(double amount))completion
     NS_SWIFT_NAME(pendingContributionsTotal(completion:));
 
-/// Links a desktop brave wallet given some payment ID
-- (void)linkBraveWalletToPaymentId:(NSString*)paymentId
-                        completion:(void (^)(LedgerResult result,
-                                             NSString* drainID))completion
-    NS_SWIFT_NAME(linkBraveWallet(paymentId:completion:));
```

Comment:
I'm wondering if [this](https://github.com/brave/brave-ios/blob/65f165a370cdaf56ec8d383272f8f7b8dca17feb/Client/Frontend/Brave%20Rewards/Transfer/WalletTransferViewController.swift#L46) has to do something with this line.

Reply:
Yeah thats just old UI for linking the brave wallet that is no longer shown, we will clean this up on iOS side when this change reaches brave-ios